include/mbedtls/config.h

```diff
@@ -3106,9 +3106,9 @@
  * \def MBEDTLS_SHA224_C
  *
  * Enable the SHA-224 cryptographic hash algorithm.
- * Future version will have this option disabled by default.
+ * A future version of the library will have this option disabled by default.
```

Comment:
I didn't pick up on this sooner, but we don't actually intend to disable SHA-224 by default. The default configuration generally includes all cryptographic mechanisms. It's only in the sample configs in `configs/*` that we intend to disable SHA-224 (because they're sample configs that require SHA-256 only).

Reply:
Oh. My bad, will fix tomorrow :)